start.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
-set -euo pipefail
+#set -euo pipefail
```

Comment:
You can't just comment out the error flags 😅

Reply:
Restored error flags and cleaned up argument logic so it doesn't trigger unbound variable.